playkit/src/main/java/com/kaltura/playkit/player/ExoPlayerWrapper.java

```diff
@@ -1060,6 +1067,10 @@ public boolean isPlaying() {
 
     @Override
     public PlaybackInfo getPlaybackInfo() {
+        if (bandwidthMeter == null || trackSelectionHelper == null) {
```

Comment:
@GouravSna 

we can return null in such case 
but in the PlayerController we can check if the value is null and return so the event will not be fired with null value

case PLAYBACK_INFO_UPDATED:
        -> if null return
                        event = new PlayerEvent.PlaybackInfoUpdated(player.getPlaybackInfo());
                        break;

Reply:
Done.